src/bin/server.rs

```diff
@@ -1,4 +1,31 @@
+extern crate bincode;
+extern crate rustc_serialize;
+extern crate rpsrtsrs;
```

Comment:
Actually it appears to be the right way to do this. See https://www.reddit.com/r/rust/comments/2bhr28/cargo_how_to_use_srcbin_effectively/. The alternative would be to specify the binaries and libraries explicityl in Cargo.toml.